book/src/advanced_database.md

```diff
@@ -14,7 +14,7 @@ more detail below).
 The full states upon which blocks are replayed are referred to as _snapshots_ in the case of the
 freezer DB, and _epoch boundary states_ in the case of the hot DB.
 
-The frequency at which the hot database stores full `BeaconState`s is fixed to one-state-per-epoch
+The frequency at which the hot database stores full `BeaconState` is fixed to one-state-per-epoch
```

Comment:
This should be pluralised, does the spellchecker trip on this?

Reply:
yes it does, unless we add the word to the `wordlist.txt`

Reply:
it seems to have passed CI just now?

Reply:
https://github.com/sigp/lighthouse/actions/runs/12367575564/job/34516125852?pr=6627

Reply:
Should be good then, I think it's because the word in `code block` is not checked after some extensions are added to the yml file